package.json

```diff
@@ -69,6 +69,7 @@
     "postcss-loader": "^3.0.0",
     "precss": "^4.0.0",
     "prop-types": "^15.7.1",
+    "query-string": "^6.2.0",
```

Comment:
Please don't use this package. Create a simple helper function that will do the same thing as this guy.

Reply:
I have resolved this